### ISSUE TITLE
Corrupting World when Disabled features

### ISSUE THREAD

**Issue by JasonMcRay** [amadornes/Framez#36]
I have no idea how to summarize this issue so here are the steps to reproduce:

1) Load world with default configs
2) Exit the world
3) Disable for example RF API support (motors)
4) Try to load the world now.
5) This screen will welcome you saying that the world is completely broken:
![image](https://cloud.githubusercontent.com/assets/4236895/11319579/179cbe26-907c-11e5-94ed-e54259934fb9.png)
6) When everything is set back or Framez 2 is removed, everything loads fine

Log: https://paste.ee/p/wFJJH

**Comment by amadornes** [comment]
Yes... this is a thing... Quick tip for other modders: Don't add multiple entries for the same block to the missing blocks map...
I was trying to replace all old/missing blocks with air and failed horribly at it. Removing that code fixes it. I'll try to push an update today that fixes it.
Also, don't worry about your world, it isn't lost. Putting in the new version will allow you to join :)

**Comment by AEnterprise** [comment]
@amadornes there is no need to set old blocks to air, forge handles that, if a block is missing and not remaped forge replaces it with air

**Comment by amadornes** [comment]
I know, but I didn't want the message to show up. I also "try" to replace motors, when disabled, with another variant so they don't just vanish. That also seems to cause trouble....
Anyway, it should be fixed now. I'll have a look through the code to see if I've missed anything that shouldn't be there in a bit and release an update :)

**Comment by AEnterprise** [comment]
i think it's good that the message pops up so people know stuff will vanish
